### PR TITLE
Fixes for unblocking RHEL-10 container builds

### PR DESCRIPTION
**Pull request by KKoukiou**
This will unblock container refreshes.
This affects also RHEL.

Related: https://bugzilla.redhat.com/show_bug.cgi?id=2295428
Related: https://issues.redhat.com/browse/RHEL-46558

Cherry-picked from master branch commit: 56e02693154ff7c1d91ae914592e18cbd9bb3ab7.

### CODE REVIEW

File: tests/unit_tests/pyanaconda_tests/modules/storage/partitioning/test_module_scheduler.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def test_get_default_file_system(self):
     def test_get_supported_raid_levels(self):
         """Test GetSupportedRaidLevels."""
         assert self.interface.GetSupportedRaidLevels(DEVICE_TYPE_MD) == \
-            ['linear', 'raid0', 'raid1', 'raid10', 'raid4', 'raid5', 'raid6']
```
Comment (M4rtinK) [inline tests/unit_tests/pyanaconda_tests/modules/storage/partitioning/test_module_scheduler.py:172]: Is this intended to be there ? I find it strange this is part of an infra PR ?

**Review by M4rtinK — APPROVED** [review]
Looking good, just not sure if the second commit is intended to be actually a part of this infra PR ?

File: tests/unit_tests/pyanaconda_tests/modules/storage/partitioning/test_module_scheduler.py (hunk 1 above)
Comment (KKoukiou) [reply tests/unit_tests/pyanaconda_tests/modules/storage/partitioning/test_module_scheduler.py:172]: Well it's not an infra PR.
It's a PR with some fixes that are needed to re-enable building centos10 and centos9 containers

Comment (KKoukiou) [reply tests/unit_tests/pyanaconda_tests/modules/storage/partitioning/test_module_scheduler.py:172]: Even the 'infra' marked commit - has tests changes.

**Comment by KKoukiou** [comment]
infra check is naive - checking per-pr - not per-commit validity.

**Pull request merged**